content/en/docs/reference/command-line-tools-reference/feature-gates/pod-level-resources.md

```diff
@@ -0,0 +1,13 @@
+---
+title: PodLevelResources
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+Enables Pod Level Resources specifications.
```

Comment:
(nit)
```suggestion
Enable _Pod level resources_:  the ability to specify resource requests and limits at the Pod
level, rather than only for specific containers.
```